app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LoginActivity">
+
+    <ImageView
+        android:id="@+id/chatair_logo"
+        android:layout_width="138dp"
+        android:layout_height="146dp"
```

Comment:
Why is this hard-coded? Set constraints here.

Reply:
Made adjustments [here](https://github.com/ChatAir/ChatAir-android/pull/16/commits/44fb9c9bcee9af8e499aae49ab7f23bb7a219a0e)